validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import eeraster
</pre><p>we can make the ingestion automatic, but only to the development environment. we will want a separate more manual routine for updating production.</p><p>Where do you pass in the destination path in earth engine?  dev env is "river-sky..." whereas production is "timberid-prd"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
The default arg is the dev path. I suppose I can make it explicit.